### PR TITLE
:sparkles: Add --yes flag

### PR DESCRIPTION
**Pull request by NikxDa**
Here's the synced up version. WIP with the features discussed. I'll update accordingly!
Reference: #44, #40

### CODE REVIEW

**Comment by NikxDa** [comment]
@kefranabg Please have a look. I couldn't get the `inquirer.prompt` test to work because the `askQuestions` call is mocked, so I left that part out. All the other tests should be useful though.

**Comment by kefranabg** [comment]
I don't see any test that is failing. if you removed it, could you add it again so I can fix it ?

File: src/cli.js [hunk 1]
```diff
@@ -33,6 +36,22 @@ const askQuestions = async projectInfos => {
...
+ * Get the default answer depending on the question type
+ *
+ * @param {Object} question
+ */
+const getDefaultAnswer = question => {
+  switch (question.type) {
+    case 'input':
+      return question.default || ''
+    case 'checkbox':
+      return question.choices.filter(itm => itm.checked).map(itm => itm.value)
```
Comment (kefranabg) [inline src/cli.js]:
```suggestion
      return question.choices.filter(choice => choice.checked).map(choice => choice.value)
```

**Comment by NikxDa** [comment]
> I don't see any test that is failing. if you removed it, could you add it again so I can fix it ?

It's not done. If you could go ahead and add a test to make sure `inquirer.prompt` is not fired, this should be done. 👍🏻

File: src/index.js [hunk 2]
```diff
@@ -16,6 +16,9 @@ yargs
       }),
     args => mainProcess(args)
   )
+  .boolean('yes')
+  .alias('y', 'yes')
+  .describe('yes', 'Use default values for all fields.')
```
Comment (kefranabg) [inline src/index.js]:
```suggestion
  .describe('yes', 'Use default values for all fields')
```

**Pull request merged**